Samples/UIWidgetsSamples_2019_4/Assets/WidgetsSample/MobileTouchSample.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
```

Comment:
Adding the meta file just to keep consistence with other files in the same folder.